test/docker_migration_test_step_2.sh

```diff
@@ -0,0 +1,145 @@
+set -eo pipefail
```

Comment:
These two files are temporarily there with the PR, just for testing purpose.
I have taken following steps for testing:
1. Set `currentMajorSchemaVersion` to 3
2. Run test/docker_migration_test_step_1.sh to create db with older db_schema version.
3. Set `currentMajorSchemaVersion` to 4
4. Run test/docker_migration_test_step_2.sh

There seems some issue with my testing, sometime it creates random names for local-gateway container, but if I do not kill the containers, I can see on crdb console that database gets renamed to `rid` from `defaultdb`.